<?xml version="1.0"?>
<launch>
    <arg name="script" default="nav_cloning_node.py"/>
    <node pkg="nav_cloning" type="$(arg script)" name="nav_cloning_node" output="screen"/>
</launch>
